ext/nnstreamer/tensor_filter/tensor_filter_nnfw.c

```diff
@@ -649,12 +649,26 @@ nnfw_invoke (const GstTensorFilterProperties * prop,
   return err;
 }
 
+/**
+ * @brief Check support of the backend
+ * @param hw: backend to check support of
+ */
+static int
+nnfw_checkAvailability (accl_hw hw)
+{
+  if (g_strv_contains (nnfw_accl_support, get_accl_hw_str (hw)))
+    return 0;
+
+  return -ENOENT;
+}
+
 static gchar filter_subplugin_nnfw[] = "nnfw";
 
 static GstTensorFilterFramework NNS_support_nnfw = {
   .version = GST_TENSOR_FILTER_FRAMEWORK_V0,
   .open = nnfw_open,
   .close = nnfw_close,
+  .checkAvailability = nnfw_checkAvailability,
```

Comment:
callback for GstTensorFilterFramework_v0
recommended adding this in init_filter_nnfw()
```
NNS_support_nnfw.checkAvailability = nnfw_checkAvailability;
```

Reply:
Thank you! Updated.